scripts/devenv-builder/configure-vm.sh

```diff
@@ -47,6 +47,15 @@ if [ $RHEL_SUBSCRIPTION = true ] ; then
     fi
 fi
 
+# FIXME: Temporarily disable selinux until we update the version of
+# RHEL used in CI.
+if ! grep -q 'Red Hat Enterprise Linux release 9.2' /etc/redhat-release; then
+    echo "WARNING: disabling selinux on $(cat /etc/redhat-release)"
+    sudo setenforce Permissive
+    sudo sed --in-place -e '/SELINUX=enforcing/d' /etc/selinux/config
+    echo "SELINUX=permissive" | sudo tee -a /etc/selinux/config
```

Comment:
We usually run this script many times on the same dev host, so we should keep it idempotent.
I'd suggest replacing SELINUX setting in one line, rather than appending to the file.
```
sed -i '/SELINUX=.*/SELUNUX=permissive/g/' /etc/selinux/config
```

Reply:
Done